### PR TITLE
Improve session hanling in admin tests

### PR DESCRIPTION
**Pull request by arnaud-morvan**
This avoid any problem with session instances expiration in admin tests.

### CODE REVIEW

File: admin/acceptance_tests/conftest.py [hunk 1]
```diff
@@ -21,15 +21,16 @@ def dbsession(settings):
     init_db(engine, force=True)
     session_factory = get_session_factory(engine)
     session = get_tm_session(session_factory, transaction.manager)
-    return session
+    yield session
```
Comment (sbrunner) [inline admin/acceptance_tests/conftest.py]: Just for info, does it change something?

Comment (arnaud-morvan) [reply admin/acceptance_tests/conftest.py]: No, but I've used a profiler here, and I've live it like this

Comment (sbrunner) [reply admin/acceptance_tests/conftest.py]: OK, thanks :-)

**Comment by arnaud-morvan** [comment]
warnings have been fixed in c2cgeoform, needs to upgrade.
@sbrunner : New issue in jenkings seems related to ngeo. Would appreciate a pinned version of ngeo.

**Comment by sbrunner** [comment]
This one will aspires event if we pin ngeo because OpenLayers don't pin one of his important dependency ..., We are working on it to fix it...

**Pull request merged**